server/notification-providers/dingding.js

```diff
@@ -11,17 +11,22 @@ class DingDing extends NotificationProvider {
      */
     async send(notification, msg, monitorJSON = null, heartbeatJSON = null) {
         const okMsg = "Sent Successfully.";
-
+        const mentionAll = notification.mentioning === "everyone";
+        const mobileList = notification.mentioning === "specify-mobiles" ? notification.mobileList.split(",") : [];
+        const userList = notification.mentioning === "specify-users" ? notification.userList.split(",") : [];
+        const mentionStr = [ ...mobileList || [], ...userList || [] ].map(item => `@${item}`).join(" ");
         try {
             if (heartbeatJSON != null) {
                 let params = {
                     msgtype: "markdown",
                     markdown: {
                         title: `[${this.statusToString(heartbeatJSON["status"])}] ${monitorJSON["name"]}`,
-                        text: `## [${this.statusToString(heartbeatJSON["status"])}] ${monitorJSON["name"]} \n> ${heartbeatJSON["msg"]}\n> Time (${heartbeatJSON["timezone"]}): ${heartbeatJSON["localDateTime"]}`,
+                        text: `## [${this.statusToString(heartbeatJSON["status"])}] ${monitorJSON["name"]} \n> ${heartbeatJSON["msg"]}\n> Time (${heartbeatJSON["timezone"]}): ${heartbeatJSON["localDateTime"]}${"\n\n" + mentionStr}`,
```

Comment:
If we notify users below (or does `atUserIds`/.. work differently), does adding this to the notification still make sense?

Reply:
Yes, the documentation clearly states that you need to add `@user` to the text field.

![image](https://github.com/user-attachments/assets/61623a15-a36a-422d-94c1-a59c67c55bc6)

Reply:
Seems weird, but ok